src/mjolnir/ingest_transit.cc

```diff
@@ -699,20 +700,20 @@ void stitch_tiles(const boost::property_tree::ptree& pt,
       // do while we have more to find and arent sick of searching
       std::set<GraphId, dist_sort_t> unchecked(all_tiles.cbegin(), all_tiles.cend(),
                                                dist_sort_t(current, grid));
+      // the first one will have to be the same tile as current, so remove it
+      unchecked.erase(unchecked.begin());
```

Comment:
the `current` tile will always be the first one, we can remove it here instead of adding a conditional further down